src/api/N2Go_Api.php

```diff
@@ -62,49 +62,42 @@ public static function run()
     private static function getPost($id)
     {
         global $wpdb;
+
+        /** @var WP_Post $post */
         $post = $wpdb->get_row(
             $wpdb->prepare("
                 SELECT 
-                    p.ID as ID, 
-                    p.guid as guid, 
-                    p.post_content as description,
-                    p.post_excerpt as shortDescription,
-                    p.post_title as title, 
-                    p.post_date as date,
-                    u.display_name as author 
+                    p.ID,
+                    p.post_content,
```

Comment:
Note: by using the raw DB columns, we get object type-hinting from WP_Post for free! 😃